lib/client-assets.php

```diff
@@ -774,6 +774,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 
 	gutenberg_extend_wp_api_backbone_client();
 
+	wp_enqueue_script( 'heartbeat' );
```

Comment:
Could help to leave a comment why we're enqueuing this separately (as an optional dependency).